src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/IndirectionNode.cs

```diff
@@ -13,18 +15,21 @@ namespace ILCompiler.DependencyAnalysis
     public class IndirectionNode : ObjectNode, ISymbolNode
     {
         private ISymbolNode _indirectedNode;
+        private int _offsetDelta;
         private TargetDetails _target;
 
-        public IndirectionNode(TargetDetails target, ISymbolNode indirectedNode)
+        public IndirectionNode(TargetDetails target, Tuple<ISymbolNode, int> indirectionData)
```

Comment:
Can you just make the constructor accept two parameters? If someone opens this file without any context, they'll just be puzzled why the signature looks like this. (E.g. why it's not `Tuple<TargetDetails, ISymbolNode, int>`?) The fewer WTF moments people have when reading code, the more readable the code is.